CONTRIBUTING.md

```diff
@@ -197,14 +197,16 @@ We have multiple test categories within packit-service:
 
 ### Running tests locally
 
-You can run unit and integration tests locally in a container:
+You can run unit and integration tests locally in a container;
+make sure that you have the ansible and podman packages installed,
```

Comment:
Is ansible relly required?

Reply:
I think ansible is only required by check-inside-openshift, check-inside-openshift-zuul,
and setup-inside-toolbox, so I've removed the mention from CONTRIBUTING.md.